test/lib/npm.js

```diff
@@ -505,50 +481,28 @@ t.test('timings', async t => {
   })
 
   const timingDisplay = [
-    [{ loglevel: 'silly' }, true, false],
+    [{ loglevel: 'silly', timing: false }, true, false],
     [{ loglevel: 'silly', timing: true }, true, true],
     [{ loglevel: 'silent', timing: true }, false, false],
   ]
 
   for (const [config, expectedDisplay, expectedTiming] of timingDisplay) {
     const msg = `${JSON.stringify(config)}, display:${expectedDisplay}, timing:${expectedTiming}`
     await t.test(`timing display: ${msg}`, async t => {
-      const { display } = await loadMockNpm(t, { config })
-      t.equal(!!display.length, expectedDisplay, 'display')
-      t.equal(!!display.timing.length, expectedTiming, 'timing display')
+      const { logs } = await loadMockNpm(t, { config })
```

Comment:
Is this testing something fundamentally different now?

Reply:
`display` was a hacky attempt at getting logs filtered by whether they would have been displayed to the user. Now we get that by only mocking `stdout` and `stderr` so it's safe to assert these on `logs`.